src/Datadog.Trace/Tracer.cs

```diff
@@ -440,19 +440,19 @@ private static string GetApplicationName()
 
         private static IStatsd CreateDogStatsdClient(TracerSettings settings, string serviceName, int port)
         {
-            var frameworkDescription = FrameworkDescription.Create();
-
-            string[] constantTags =
+            try
             {
-                "lang:.NET",
-                $"lang_interpreter:{frameworkDescription.Name}",
-                $"lang_version:{frameworkDescription.ProductVersion}",
-                $"tracer_version:{TracerConstants.AssemblyVersion}",
-                $"service_name:{serviceName}"
-            };
+                var frameworkDescription = FrameworkDescription.Create();
+                string[] constantTags = { "lang:.NET", $"lang_interpreter:{frameworkDescription.Name}", $"lang_version:{frameworkDescription.ProductVersion}", $"tracer_version:{TracerConstants.AssemblyVersion}", $"service_name:{serviceName}" };
```

Comment:
Can you have one tag per line as in the old code? It's easier to read.

Reply:
Sorry, "when resharper attacks"
Will fix.

Reply:
Thanks for reverting it. I recommend inspecting your diffs before committing them and reviewing your own PR to avoid changes like these.